docs/getting-started/form-submissions.md

```diff
@@ -71,6 +72,42 @@ query set as the `q` query parameter in the URL. The `GET` handler will then
 filter the names array based on the query, and pass it to the page component for
 rendering.
 
+## `POST` example
+Here's an example processing a simple form submission request:
+
+```tsx
+// routes/subscribe.tsx
+import { Handlers } from "$fresh/server.ts";
+
+export const handler: Handlers = {
+  async GET(req, ctx) {
+    return await ctx.render();
+  },
+  async POST(req, ctx) {
+    try {
+      const form = await req.formData();
+      const email = form.get("email");
+      return new Response(JSON.stringify({ email }));
```

Comment:
There should be a check here that the returned value exists, and is a string (as opposed to being a `File`).